File: remote/src/main/java/com/kelsos/mbrc/events/ui/VolumeChange.java
package com.kelsos.mbrc.events.ui;

public class VolumeChange {
  private int volume;
  private boolean mute;

  public VolumeChange(int vol) {
    this.volume = vol;
    this.mute = false;
  }

  public VolumeChange() {
    this.volume = 0;
    this.mute = true;
  }

  public int getVolume() {
    return this.volume;
  }

  public boolean isMute() {
    return this.mute;
  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/events/ui/VolumeChange.kt
package com.kelsos.mbrc.events.ui

class VolumeChange {
  var volume: Int = 0
    private set
  var isMute: Boolean = false
    private set

  constructor(vol: Int) {
    this.volume = vol
    this.isMute = false
  }

  constructor() {
    this.volume = 0
    this.isMute = true
  }
}
